book/src/usage.md

```diff
@@ -93,40 +117,36 @@ a more intuitive tree format:
    └──────────┘  └──────────┘  └──────────┘
 ```
 
-Say we have a selection that wraps `arg1`. The selection is on the `arg1` leaf
-in the tree above.
-
-```
-func([arg1], arg2, arg3)
-```
-
-Using `Alt-n` would select the next sibling in the syntax tree: `arg2`.
+If you have a selection that wraps `arg1` (see the tree above), and you use
+Alt-n, it will select the next sibling in the syntax tree: `arg2`.
 
-```
-func(arg1, [arg2], arg3)
+```js
+func([arg1], arg2, arg3) > func(arg1, [arg2], arg3);
```

Comment:
```suggestion
// before
func([arg1], arg2, arg3)
// after
func(arg1, [arg2], arg3);
```

since it's a code block I think the `>` could throw off syntax highlighting or be visually confusing